docs/conf.py

```diff
@@ -98,7 +98,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = []
```

Comment:
Just curious, why did you delete the `_static` dir?

Reply:
Because you are not using it and is causing warnings during build of documentation:

https://github.com/neovim/python-client/pull/317/commits/9e8a9d07a02f7ac3a82fcccc6919f3414611da0d

```
WARNING: html_static_path entry u'docs/_static' does not exist
```